data/avgi/avgi outfits.txt

```diff
@@ -0,0 +1,1488 @@
+
+
+outfit "Diffuse Deflector"
+	category "Systems"
+	licenses
+		"Avgi Civilian"
+	cost 1200000
+	thumbnail "outfit/unknown"
+	"mass" 48
+	"outfit space" -48
+	"shield generation" 0.6
+	"shield energy" 1.8
+	"shield heat" 4.8
+	"shield delay" 3
+	description "For its size, you would expect this shield generator to be far more powerful than it actually is. While it is nearly as large as the heaviest human shield generators, it is only about three quarters as strong, while also consuming much more power. Much of this power ends up being shed as waste heat instead of being used to efficiently regenerate a shield."
+
+
+outfit "Specular Deflector"
+	category "Systems"
+	licenses
+		"Avgi Military"
+	cost 2100000
+	thumbnail "outfit/unknown"
+	"mass" 72
+	"outfit space" -72
+	"shield generation" 1.2
+	"shield energy" 3.6
+	"shield heat" 9.6
+	"shield delay" 3
+	description "The largest shield generator manufactured by the Avgi is impressive, if only because of how far the Avgi have pushed their limited theory of shielding principles. While their engineering ingenuity is admirable, there is a limit to how far you can take an incomplete understanding of the base physics behind shielding, and this generator approaches that limit."
+
+
+outfit "Optical Lasing Generator"
+	category "Systems"
+	licenses
+		"Avgi Civilian"
+	cost 120000
+	thumbnail "outfit/unknown"
+	"mass" 30
+	"outfit space" -30
+	"lasing power" 6
+	description "This coil of optical fiber is dozens of kilometers long, tightly wrapped around the generator that pumps the lasing medium with the power needed to produce an intense laser beam. The produced laser light, which starts in the infrared range, can be easily tapped and routed via optical fiber to a number of turrets on a ship, eliminating the need for the heavy lasing generator to be anywhere near the turret itself."
+	description "The infrared light produced in the coil can either be sent through a frequency doubling crystal to support up to three green Phased Array Lasers, or frequency tripled to support up to two blue Phasers."
+
+
+
+# Power:
+
+
+outfit "Concentrating Solar Cell"
+	category "Power"
+	licenses
+		"Avgi Civilian"
+	cost 200000
+	thumbnail "outfit/unknown"
+	"mass" 1
+	"outfit space" -3
+	"solar collection" 0.3
+	"solar heat" 0.12
+	description "This tiny set of concentrating solar cells efficiently collects sunlight, turning it into usable energy. Thin Fresnel lenses concentrate sunlight onto a small number of highly efficient solar cells, reducing weight and improving efficiency."
+
+
+outfit "Concentrating Solar Array"
+	category "Power"
+	licenses
+		"Avgi Civilian"
+	cost 200000
+	thumbnail "outfit/unknown"
+	"mass" 3
```

Comment:
I'm not entirely sure how to evaluate the balance implications of outfits having lower than standard densities. I recall noting recently that most ships use around 10-20% of their outfit space on generators, so if you use exclusively these for power generation you're losing *very* roughly 5%-15% of your ship's mass, depending on its ratio of dry mass to outfit space (which I'm assuming to mostly vary from .25 to 1).
In terms of power per space though, this and the Concentrating Solar Cell are notably weaker (~10%-20%) than coalition collector modules, and I don't think the mass saving is likely to be so overpowered as to cause problems there. Exclusive reliance on solar panels carries additional difficulties too, so I think these are reasonable. Might lead to interesting fighter builds, too.